Ibidapo/send-it
# #162770365 Implements email notification using SendGrid

#### What does this PR do?
- Implements email notification after creating or updating an existing parcel
- Refactored Forgot Password email notification

#### How should this be manually tested?
- Log onto to the application https://ibidapo.github.io/send-it/UI/
-  As a new user, register with valid credentials
- As an existing user, login with valid credentials
- Create or update an existing parcels
- Then check your mail for notification

#### What are the relevant pivotal tracker stories?
#162770365
